Translations update from Hosted Weblate

weblate opened the pull request:
Translations update from [Hosted Weblate](https://hosted.weblate.org) for [Setzer/Application](https://hosted.weblate.org/projects/setzer/application/).



Current translation status:

![Weblate translation status](https://hosted.weblate.org/widgets/setzer/-/application/horizontal-auto.svg)

nbehrnd committed:
Translated using Weblate (German)

Currently translated at 97.5% (586 of 601 strings)

Translation: Setzer/Application
Translate-URL: https://hosted.weblate.org/projects/setzer/application/de/

oscfdezdz commented:
@nbehrnd you can add yourself to the credits if you want, the string is `translator-credits`.

nbehrnd committed:
Translated using Weblate (German)

Currently translated at 97.5% (586 of 601 strings)

Translation: Setzer/Application
Translate-URL: https://hosted.weblate.org/projects/setzer/application/de/

nbehrnd commented:
@stephanlachnit, @cvfosammmm  Especially around the topic to create/curate the .bib file, there are a few strings to be polished, or to be amended.*  I'm not aware if Setzer (would) adopt(ed) an approach similar to the one around Python editor [Thonny](https://github.com/thonny/thonny/blob/master/docs/translate.md), i.e. while the translation to e.g. German and French was still ongoing, the not yet translated English strings were used *as such* to fill the gaps when running the program with the French/the German profile of the interface.

Question/request: can one of you have look on them to complete the set of strings?

\* And there equally are two on the welcome screen which should be attractive, too.

Giai-Ngo committed:
Added translation using Weblate (Vietnamese)

Giai-Ngo committed:
Translated using Weblate (Vietnamese)

Currently translated at 98.0% (589 of 601 strings)

Translation: Setzer/Application
Translate-URL: https://hosted.weblate.org/projects/setzer/application/vi/

nbehrnd committed:
Translated using Weblate (German)

Currently translated at 99.5% (598 of 601 strings)

Translation: Setzer/Application
Translate-URL: https://hosted.weblate.org/projects/setzer/application/de/

nbehrnd committed:
Translated using Weblate (German)

Currently translated at 99.6% (599 of 601 strings)

Translation: Setzer/Application
Translate-URL: https://hosted.weblate.org/projects/setzer/application/de/

nbehrnd committed:
Translated using Weblate (German)

Currently translated at 99.8% (600 of 601 strings)

Translation: Setzer/Application
Translate-URL: https://hosted.weblate.org/projects/setzer/application/de/

lumingzh committed:
Translated using Weblate (Chinese (Simplified))

Currently translated at 95.6% (575 of 601 strings)

Translation: Setzer/Application
Translate-URL: https://hosted.weblate.org/projects/setzer/application/zh_Hans/

cvfosammmm committed:
recolor help panel

turtlegarden committed:
Added translation using Weblate (Norwegian Nynorsk)

Cyan-Coder committed:
Added translation using Weblate (Dutch)

demirdagli1 committed:
Added translation using Weblate (Turkish)

eliotttak committed:
Added translation using Weblate (French)

NorwayFun committed:
Added translation using Weblate (Georgian)

milotype committed:
Added translation using Weblate (Croatian)
